### PR TITLE
tests

### PR DESCRIPTION
**Pull request by ntotomanov-taulia**
@brentfisher-taulia please review :)

### CODE REVIEW

File: init/src/main/groovy/com/taulia/boundaries/Transmission.groovy [hunk 1]
```diff
@@ -4,7 +4,15 @@ package com.taulia.boundaries
...
   Gear gear
   Moveable moveable
-  Transmission(Moveable moveable){
+
+  Transmission(Moveable moveable) {
     this.moveable = moveable
   }
+
+  def shift(Gear gear) {
+    if (this.gear == Gear.DRIVE && this.moveable.currentSpeedInMph() != 0 && gear == Gear.PARK) {
```
Comment (brentfisher-taulia) [inline init/src/main/groovy/com/taulia/boundaries/Transmission.groovy:13]: would this ever work?  Should it be an or?  Is there a test for both conditions?

**Comment by brentfisher-taulia** [comment]
Can you finish time traveller test?

File: init/src/main/groovy/com/taulia/boundaries/Transmission.groovy (hunk 1 above)
Comment (brentfisher-taulia) [reply init/src/main/groovy/com/taulia/boundaries/Transmission.groovy:13]: oh, wait, I see.  nevermind.

Comment (ntotomanov-taulia) [reply init/src/main/groovy/com/taulia/boundaries/Transmission.groovy:13]: its ugly, I know :) I don't like it too

**Pull request closed without merging**